aopy/visualization/base.py

```diff
@@ -1702,7 +1703,7 @@ def plot_boxplots(data, plt_xaxis, trendline=True, facecolor='gray', linecolor='
             boxprops=dict(facecolor=facecolor, color=linecolor), capprops=dict(color=linecolor),
             whiskerprops=dict(color=linecolor), flierprops=dict(color=facecolor, markeredgecolor=facecolor),
             medianprops=dict(color=linecolor))
-    ax.set_xticklabels(plt_xaxis)
+    ax.set_xticklabels(plt_xaxis) if plt_xaxis else None
```

Comment:
this is a bit hard to read, could you replace with
```suggestion
    if label_xticks:
        ax.set_xticklabels(plt_xaxis)
```